model/pytorch/dffml_model_pytorch/utils/utils.py

```diff
@@ -34,6 +33,45 @@ def create_layer(layer_dict):
     return sequential_dict
 
 
+def create_network(network_dict):
```

Comment:
This function is basically taking an dict(in the form of https://github.com/intel/dffml/blob/master/model/pytorch/examples/resnet18/layers.yaml) which contains a model to be trained , and then its converting to a `Sequential model` which will eventually be converted as a `PytorchNeuralNetwork` [like we have this now](https://github.com/intel/dffml/blob/42bb2aa101562ca6dbaeded6acf1125edbdcfe27/model/pytorch/tests/test_pytorchnet.py#L65). Moreover I saw that presently if we are passing a dict of the models instead of `nn.Sequential` we were getting an error from the `test cases`

Reply:
Also, instead of creating a completely new function, we should be updating the already existing function `create_layer`

Reply:
@sakshamarora1 just wanted to be sure , actually I was getting an error when I was passing a dict model to that class (In test currently we are having only the tests with `nn` models), have you also also faced the same error , or is it an issue from my side only?